docs/configuration-reference/components/cluster-autoscaler.md

```diff
@@ -83,7 +83,7 @@ Example:
 | `provider`                   | Supported provider, currently Packet.                                                    | "packet"      | false    |
 | `packet.project_id`          | Packet Project ID where the cluster is running.                                          | -             | true     |
 | `packet.facility`            | Packet Facility where the cluster is running.                                            | -             | true     |
-| `packet.worker_type`         | Machine type for workers spawned by the Cluster Autoscaler.                              | "baremetal_0" | false    |
```

Comment:
Can we please align table columns? New default value is longer than the previous one. They then look nicer while editing in plain editor.

Reply:
use this to format http://markdowntable.com/